### PR TITLE
Tvlatas/vz 2575

### PR DESCRIPTION
**Pull request by agv426**
# Description

Change image pull policies where possible to be using IfNotPresent instead of Always, and also for the integ-tests have the application operator use the local chart as well.

This should significantly reduce the image pulls during tests from ghcr.io and use the pre-pulled and cached images. This might help with ghcr.io rate limites, but it should at least help with test times/resiliency using the caches more.

# Checklist 

As the author of this PR, I have:

- [X] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: application-operator/installer/scripts/1-install-oam-runtime.sh [hunk 1]
```diff
@@ -29,10 +29,11 @@ function install_oam {
   log "Install OAM"
   helm install oam \
       --namespace "${NAMESPACE}" \
-      --set image.repostiory=ghcr.io/verrazzano/oam-kubernetes-runtime \
+      --set image.repository=ghcr.io/verrazzano/oam-kubernetes-runtime \
```
Comment (gilbode) [inline application-operator/installer/scripts/1-install-oam-runtime.sh:32]: image.repository and image.tag are no longer needed if you are using the values file b/c they're specified in there.

**Review by gilbode — APPROVED** [review]

File: application-operator/installer/scripts/1-install-oam-runtime.sh (hunk 1 above)
Comment (agv426) [reply application-operator/installer/scripts/1-install-oam-runtime.sh:32]: doh, yeah missed removing those. I can update those and re-push/test so cleaning that out doesn't get lost

**Review by bobdonat — COMMENTED** [review]
A general comment.  We should not be changing values.yaml in the thirdparty helm charts but instead specifying the new values in the override files that are passed to the helm charts.

**Comment by agv426** [comment]
Taking a pass on the 3rdparty changes

File: application-operator/installer/scripts/common.sh [hunk 2]
```diff
@@ -12,6 +12,9 @@ BUILD_DIR="${SCRIPT_DIR}/build"
 # The max length of the environment name passed in by the user.
 ENV_NAME_LENGTH_LIMIT=10
 
+CHARTS_DIR=$(cd $SOURCE_DIR/../../../platform-operator/thirdparty/charts; pwd -P)
```
Comment (pfmackin) [inline application-operator/installer/scripts/common.sh:15]: we already have CHARTS_DIR  and VZ_OVERRIDES_DIR in common.sh

CHARTS_DIR=$(cd $SOURCE_DIR/../../thirdparty/charts; pwd -P)
VZ_CHARTS_DIR=$(cd $SOURCE_DIR/../../helm_config/charts; pwd -P)
VZ_OVERRIDES_DIR=$(cd $SOURCE_DIR/../../helm_config/overrides; pwd -P)

Comment (agv426) [reply application-operator/installer/scripts/common.sh:15]: there appear to be multiple common.sh files, this one was for the application-operator common.sh file

**Review by kminder — APPROVED** [review]
Approved.

**Pull request merged**